backend/src/main/resources/db/migration/V2__Event.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE event(
+    id INT AUTO_INCREMENT PRIMARY KEY,
+	name varchar(256),
+	description TEXT,
+	date Date,
+    is_required BOOLEAN DEFAULT true
+);
+
+INSERT INTO event (name, description, date)
+VALUES  ('Major/Minor & Career Exploration Event', 'Explore careers', '2024-04-03'),
+        ('Involvement Fair', 'get involved', '2024-04-03'),
+        ('Major/Minor & Career Exploration Event', 'Explore careers', '2025-04-03'),
+        ('Networking & Professionalism Workshop', '(online session)', '2025-04-03'),
+        ('Attend Job Fair Fall Semester', 'Explore jobs', '2025-04-03'),
+        ('Attend Job Fair Spring Semester', 'Explore jobs', '2025-04-03'),
+        ('(Choose from list of School of Business Events) ', 'choose one', '2025-04-03'),
```

Comment:
is the student meant to select an event or something? or is this just mock data that can be anything?

Reply:
This is the mock data they gave us for events, These were supposed to be the "task to attend an event", but this is also just mock data for testing purposes